app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -79,13 +78,6 @@ def destroy
 
   private
 
-  def rename_config_template
-    if params[taxonomy_single] && params[taxonomy_single][:config_template_ids].present?
-      params[taxonomy_single][:provisioning_template_ids] = params[taxonomy_single].delete(:config_template_ids)
-      Foreman::Deprecation.deprecation_warning('1.11', 'Config templates were renamed to provisioning templates')
```

Comment:
Sorry, I don't think this should have had a version set since it's a user-facing API - the rest of the API isn't set to a specific version for deprecation.